articles/communication-services/concepts/call-automation/call-automation.md

```diff
@@ -103,24 +102,24 @@ After your application has played an audio prompt, you can request user input to
 When your application answers a call or places an outbound call to an endpoint, that call can be transferred to another destination endpoint. Transferring a 1:1 call will remove your application's ability to control the call using the Call Automation SDKs.
 
 **Record**
-You decide when to start/pause/resume/stop recording based on your application business logic, or you can grant control to the end user to trigger those actions. To learn more, view our [concepts](./../voice-video-calling/call-recording.md) and [quickstart](../../quickstarts/voice-video-calling/get-started-call-recording.md).
+You can decide when to start/pause/resume/stop recording based on your application's business logic, or you can grant control to the end user to trigger those actions. To learn more, view our [concepts](./../voice-video-calling/call-recording.md) and [quickstart](../../quickstarts/voice-video-calling/get-started-call-recording.md).
 
 **Hang-up**
 When your application has answered a one-to-one call, the hang-up action will remove the call leg and terminate the call with the other endpoint. If there are more than two participants in the call (group call), performing a ‘hang-up’ action will remove your application’s endpoint from the group call.
 
 **Terminate**
-Whether your application has answered a one-to-one or group call, or placed an outbound call with one or more participants, this action will remove all participants and end the call. This operation is triggered by setting `forEveryOne` property to true in Hang-Up call action.
+Whether your application has answered a one-to-one or group call, or placed an outbound call with one or more participants, this action will remove all participants and end the call. This operation is triggered by setting the `forEveryOne` property to true in the Hang-Up call action.
 
 **Cancel media operations** 
-Based on business logic your application may need to cancel ongoing and queued media operations. Depending on the media operation canceled and the ones in queue, you will received a webhook event indicating that the action has been canceled. 
+Depending on your application's business logic, you may need to cancel ongoing and queued media operations. Depending on the media operation canceled and the ones in queue, you will receive a webhook event indicating that the action has been canceled. 
 
 ## Events
 
-The following table outlines the current events emitted by Azure Communication Services. The two tables below show events emitted by Event Grid and from the Call Automation as webhook events.
+The table below outlines the current events emitted by Azure Communication Services. The two tables that follow show the events emitted by Event Grid and Call Automation as webhook events.
```

Comment:
I think we can remove the first line - The table below outlines the current events emitted by Azure Communication Services
It looks redudant.